python/lambdas/copy-files-from-tdr/lambda_function.py

```diff
@@ -20,15 +20,19 @@ def lambda_handler(event, context):
         file_id = body["fileId"]
         metadata_file_id = f"{file_id}.metadata"
         source_bucket = body["bucket"]
-        files = [file_id, metadata_file_id]
-        assert_objects_exist_in_bucket(source_bucket, files)
-        validate_metadata(source_bucket, metadata_file_id)
-        file_location = copy_objects(destination_bucket, file_id, source_bucket)
-        copy_objects(destination_bucket, metadata_file_id, source_bucket)
-        potential_message_id = {key: value for key, value in body.items() if key == "messageId"}
-        sqs_body = {"id": file_id, "location": file_location}
-        sqs_body.update(potential_message_id)
-        sqs_client.send_message(QueueUrl=destination_queue, MessageBody=json.dumps(sqs_body))
+        try:
+            files = [file_id, metadata_file_id]
+            assert_objects_exist_in_bucket(source_bucket, files)
+            validate_metadata(source_bucket, metadata_file_id)
+            file_location = copy_objects(destination_bucket, file_id, source_bucket)
+            copy_objects(destination_bucket, metadata_file_id, source_bucket)
```

Comment:
Do you think it's worth splitting this `try` into two `try`s: one, from 24-28 for s3 errors and then line 29-32 for SQS, and then we could change the error name to be "S3 Error" or "SQS Error" and maybe incorporate more domain specific information in each error such as the destination_bucket or destination_queue?

Reply:
It's a good idea but `file_location` is needed in the calls to SQS so we'd need nested try blocks which is a bit odd. 
Also, we're already printing `f"Error during copy of '{s3_key}' from '{source_bucket}' to '{destination_bucket}': {e}"` in the copy_objects function if there was an error.

This is meant as a bit of a catch all "This failed" message and then why it's failed can be looked at elsewhere.